ci-operator/step-registry/ipi/conf/azure/customregion/ipi-conf-azure-customregion-ref.metadata.json

```diff
@@ -0,0 +1,15 @@
+{
+	"path": "ipi/conf/azure/customregion/ipi-conf-azure-customregion-ref.yaml",
```

Comment:
How about rename `customregion` to `custom-region`

Reply:
Thanks @jianlinliu  , I deliberately avoided '-' as normally in the repo , '-' represents folder as well , example ipi-conf-azure is /ipi/conf/azure , what do you think ? Can change to `custom-region` if `customregion` seems problematic.

Reply:
> '-' represents folder as well

Yeah, that is one case, another case is we can also name a folder name including `-` as a char, actually we already had many folder names including a `-`, e.g; `ci-operator/step-registry/ipi/conf/aws/custom-az/`.

Reply:
acked , will make the change for easy readability , thanks!